packages/components/src/number-control/index.js

```diff
@@ -151,7 +151,8 @@ export function NumberControl(
 		 */
 		if (
 			type === inputControlActionTypes.PRESS_ENTER ||
-			type === inputControlActionTypes.COMMIT
```

Comment:
I'm hesitant about this change.

In the [input control reducer](https://github.com/WordPress/gutenberg/blob/7a3e37a3e0f11e0a05eb2ab36d34427e1c0c5b4f/packages/components/src/input-control/reducer/reducer.ts), the reducers for `COMMIT` and `UPDATE` do the exact same thing, and so I wonder if the `UPDATE` action is necessary at all ?

Reply:
It does look like that, doesn't it? From the looks of it, we might be able to remove the `update()` action as well, and only use `commit()`, if I'm not missing anything?

Reply:
I went ahead and:
- removed he `UPDATE` action and every reference to it
- swapped `update()` with `commit()` in `InputControl`

[Done in https://github.com/WordPress/gutenberg/pull/39186/commits/e8b3d48cafb32f65a9dfb7b3e072f3869b9c118f]

Reply:
Thanks, those changes look good to me!